### PR TITLE
Post parse structure modification - execute on a list

### PR DESCRIPTION
**Pull request by drtimwright**
Execute on a list now using the structure modification code.

### CODE REVIEW

File: src/main/java/org/concordion/internal/command/executeCommand/modificationStrategies/ExecuteCommandListModification.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        return currentLevel == null ? 1 : currentLevel;
+    }
+
+    @Override
+    public boolean originalNodeShouldBypassExecution() {
+        return true;
+    }
+
+
+
```
Comment (nigelcharman) [inline src/main/java/org/concordion/internal/command/executeCommand/modificationStrategies/ExecuteCommandListModification.java]: Do we need the rest of this class from here down? I can delete them and the tests still pass.

File: src/main/java/org/concordion/internal/command/executeCommand/modificationStrategies/ExecuteCommandModification.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import org.concordion.api.ExampleCommandCall;
+
+import java.util.List;
+
+/**
+ * Created by tim on 30/10/16.
+ */
+public abstract class ExecuteCommandModification {
+    public abstract void performModification(CommandCall commandCall, List<ExampleCommandCall> examples, List<CommandCall> beforeExamples);
+    public abstract boolean originalNodeShouldBypassExecution();
```
Comment (nigelcharman) [inline src/main/java/org/concordion/internal/command/executeCommand/modificationStrategies/ExecuteCommandModification.java]: This method is unused

File: src/main/java/org/concordion/internal/command/executeCommand/modificationStrategies/ExecuteCommandListModification.java (hunk 1 above)
Comment (drtimwright) [inline src/main/java/org/concordion/internal/command/executeCommand/modificationStrategies/ExecuteCommandListModification.java]: Yes. That was copied and pasted from elsewhere and I forgot to delete it after rewriting it to suit the new structure.

File: src/main/java/org/concordion/internal/command/executeCommand/modificationStrategies/ExecuteCommandModification.java (hunk 2 above)
Comment (drtimwright) [inline src/main/java/org/concordion/internal/command/executeCommand/modificationStrategies/ExecuteCommandModification.java]: Deleted. This was something I refactored out to somewhere else (into the CommandCall I believe).

**Pull request merged**